cookbook/views/api.py

```diff
@@ -598,35 +598,31 @@ class ViewLogViewSet(viewsets.ModelViewSet):
     queryset = ViewLog.objects
     serializer_class = ViewLogSerializer
     permission_classes = [CustomIsOwner]
+    pagination_class = DefaultPagination
 
     def get_queryset(self):
-        self.queryset = self.queryset.filter(created_by=self.request.user).filter(space=self.request.space).all()
-        if self.request.method == 'GET':
-            return self.queryset[:5]
-        else:
-            return self.queryset
+        # working backwards from the test - this is supposed to be limited to user view logs only??
+        return self.queryset.filter(created_by=self.request.user).filter(space=self.request.space)
 
 
 class CookLogViewSet(viewsets.ModelViewSet):
     queryset = CookLog.objects
     serializer_class = CookLogSerializer
-    permission_classes = [CustomIsOwner]
+    permission_classes = [CustomIsOwner]  # CustomIsShared? since ratings are in the cooklog?
```

Comment:
currently all ratings only apply to the user itself. i think we can add a privacy setting that allows to share ratings and cook logs with other users or even completely change the behavior to always be public inside a space but that is a separate topic to look at i think

Reply:
thats why IsOWner is correct